### PR TITLE
Task J-16: add them main colors

### PR DESCRIPTION
**Pull request by artem-totality**

### CODE REVIEW

File: frontend/src/assets/css/variables.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+:root {
```
Comment (ghost) [inline frontend/src/assets/css/variables.scss:1]: import it in the `css/styles.scss` and remove everything from there

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by liza-veis — DISMISSED** [review]

File: frontend/src/assets/css/styles.scss [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import 'variables';
```
Comment (ghost) [inline frontend/src/assets/css/styles.scss]:
```suggestion
@import './variables';
```

File: frontend/src/assets/css/styles.scss [hunk 3]
```diff
@@ -1,3 +1,5 @@
...
+
 body {
```
Comment (ghost) [inline frontend/src/assets/css/styles.scss]: Remove all useless code and left only import of variables

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]
LGTM

**Review by liza-veis — APPROVED** [review]

**Pull request merged**